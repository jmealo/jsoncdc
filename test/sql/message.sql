\set ECHO none
SET client_min_messages TO error;
\t on
\x off
SELECT 'created logical replication slot' FROM pg_create_logical_replication_slot('jsoncdc', 'jsoncdc');
CREATE SCHEMA IF NOT EXISTS jsoncdc;
SET LOCAL search_path TO jsoncdc, public;
BEGIN;
SELECT 'msg1' FROM pg_logical_emit_message(true, 'jsoncdc', 'msg1');
SELECT 'msg2' FROM pg_logical_emit_message(false, 'jsoncdc', 'msg2');
SELECT 'msg3' FROM pg_logical_emit_message(true, 'jsoncdc', 'msg3');
SELECT 'msg4' FROM pg_logical_emit_message(false, 'jsoncdc', 'msg4');
SELECT 'msg5' FROM pg_logical_emit_message(true, 'jsoncdc', 'msg5');
SELECT 'msg6' FROM pg_logical_emit_message(false, 'jsoncdc', 'msg6');
SELECT 'msg7' FROM pg_logical_emit_message(true, 'jsoncdc', 'msg7');
SELECT 'žluťoučký kůň' FROM pg_logical_emit_message(false, 'jsoncdc', 'žluťoučký kůň');
COMMIT;
SELECT 'deleted logical replication slot'
  FROM pg_drop_replication_slot('jsoncdc');
DROP SCHEMA jsoncdc CASCADE;
