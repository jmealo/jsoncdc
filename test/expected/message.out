\set ECHO none
 created logical replication slot

 msg1
 { "message": message, "prefix": prefix, "transactional": false}
 msg2
 { "message": message, "prefix": prefix, "transactional": false}
 msg3
 { "message": message, "prefix": prefix, "transactional": false}
 msg4
 { "message": message, "prefix": prefix, "transactional": false}
 msg5
 { "message": message, "prefix": prefix, "transactional": false}
 msg6
 { "message": message, "prefix": prefix, "transactional": false}
 msg7
 { "message": message, "prefix": prefix, "transactional": false}
 žluťoučký kůň
 { "message": message, "prefix": prefix, "transactional": false}

 deleted logical replication slot
